ts/src/binance.ts

```diff
@@ -237,141 +237,137 @@ export default class binance extends Exchange {
-                        'margin/allPairs': 0.1,
-                        'margin/priceIndex': 1,
-                        // these endpoints require this.apiKey + this.secret
-                        'spot/delist-schedule': 10,
-                        'asset/assetDividend': 1,
-                        'asset/dribblet': 0.1,
-                        'asset/transfer': 0.1,
-                        'asset/assetDetail': 0.1,
-                        'asset/tradeFee': 0.1,
-                        'asset/ledger-transfer/cloud-mining/queryByPage': 4.0002, // Weight(UID): 600 => cost = 0.006667 * 600 = 4.0002
+                        'margin/allAssets': 1,
+                        'margin/allPairs': 1,
+                        'margin/priceIndex': 10,
+                        'spot/delist-schedule': 100,
+                        'asset/assetDividend': 10,
+                        'asset/dribblet': 1,
+                        'asset/transfer': 1,
+                        'asset/assetDetail': 1,
+                        'asset/tradeFee': 1,
+                        'asset/ledger-transfer/cloud-mining/queryByPage': 600,
                         'asset/convert-transfer/queryByPage': 0.033335,
-                        'asset/wallet/balance': 6, // Weight(IP): 60 => cost = 0.1 * 60 = 6
-                        'asset/custody/transfer-history': 6, // Weight(IP): 60 => cost = 0.1 * 60 = 6
-                        'margin/borrow-repay': 1,
+                        'asset/wallet/balance': 60,
+                        'asset/custody/transfer-history': 60,
+                        'margin/borrow-repay': 10,
                         'margin/loan': 1,
                         'margin/repay': 1,
-                        'margin/account': 1,
-                        'margin/transfer': 0.1,
-                        'margin/interestHistory': 0.1,
-                        'margin/forceLiquidationRec': 0.1,
-                        'margin/order': 1,
-                        'margin/openOrders': 1,
-                        'margin/allOrders': 20, // Weight(IP): 200 => cost = 0.1 * 200 = 20
-                        'margin/myTrades': 1,
-                        'margin/maxBorrowable': 5, // Weight(IP): 50 => cost = 0.1 * 50 = 5
-                        'margin/maxTransferable': 5,
-                        'margin/tradeCoeff': 1,
+                        'margin/account': 10,
+                        'margin/transfer': 1,
+                        'margin/interestHistory': 1,
+                        'margin/forceLiquidationRec': 1,
+                        'margin/order': 10,
+                        'margin/openOrders': 10,
+                        'margin/allOrders': 200,
+                        'margin/myTrades': 10,
+                        'margin/maxBorrowable': 50,
+                        'margin/maxTransferable': 50,
+                        'margin/tradeCoeff': 10,
                         'margin/isolated/transfer': 0.1,
```

Comment:
such occasions are not a fault in parser, but these values are leftovers (copied) from existing binance.ts as those endpoints are no longer found in docs (removed), so we just retain them as-is